libsolidity/inlineasm/AsmParser.cpp

```diff
@@ -252,7 +252,7 @@ assembly::Expression Parser::parseExpression()
 	{
```

Comment:
This should fail if in strict mode.

Reply:
This code is actually weird. It seems to turn an `Instruction` into `FunctionalInstruction` even there are no parentheses.

Reply:
Yes, this is a shortcut because is loose flavour, `callvalue()` an `callvalue` behave identical.